tests/index.ts

```diff
@@ -75,22 +74,260 @@ const FIRESTORE_DATA = {
     updateTime: '2020-04-23T12:00:27.247187Z',
   },
 };
+const SCHEDULER_DATA: SchedulerJobData = {
+  customData: 'bXkgYmFzZTY0IGRhdGE=',
+};
+const STORAGE_DATA: StorageObjectData = {
+  bucket: 'some-bucket',
+  contentType: 'text/plain',
+  crc32c: 'rTVTeQ==',
+  etag: 'CNHZkbuF/ugCEAE=',
+  generation: 1587627537231057,
+  id: 'some-bucket/folder/Test.cs/1587627537231057',
+  kind: 'storage#object',
+  md5Hash: 'kF8MuJ5+CTJxvyhHS1xzRg==',
+  mediaLink:
+    'https://www.googleapis.com/download/storage/v1/b/some-bucket/o/folder%2FTest.cs?generation=1587627537231057\u0026alt=media',
+  metageneration: 1,
+  name: 'folder/Test.cs',
+  selfLink:
+    'https://www.googleapis.com/storage/v1/b/some-bucket/o/folder/Test.cs',
+  size: 352,
+  storageClass: 'MULTI_REGIONAL',
+  timeCreated: '2020-04-23T07:38:57.230Z',
+  timeStorageClassUpdated: '2020-04-23T07:38:57.230Z',
+  updated: '2020-04-23T07:38:57.230Z',
+};
+const LOG_ENTRY_DATA: LogEntryData = {
```

Comment:
Nit: should these dummy values be in separate `.ts` (or even `.json`) files?

Reply:
One file is fine. Having JSON files would prevent using the interfaces, which are used for validation and examples for users, so I'd keep it in TS.